selfdrive/car/chrysler/fingerprints.py

```diff
@@ -612,4 +612,34 @@
       b'M2421132MB',
     ],
   },
+  CAR.DURANGO: {
+    (Ecu.combinationMeter, 0x742, None): [
+      b'68454261AD',
+      b'68471535AE',
```

Comment:
can you post dongles for all FW?

Reply:
4bb5387dcb2bb6e8
3379c85aeedc8285

Reply:
connect shows 4bb5387dcb2bb6e8's routes as green when below 40 mph, did you check if any of these can steer to zero?

Reply:
State 8 supposedly means you can steer, interesting, looks like you can steer to zero once you drive above 38 kph for the first time

![image](https://github.com/commaai/openpilot/assets/25857203/aca3edb5-531d-4bbd-b9c6-6f6d2635b513)

Reply:
`3379c85aeedc8285` only allows steering at 40 mph, and then stops steering at 35 mph though, so the higher limit for this PR is fine, we can detect this later as an enhancement